lib/exporters/jhipster_deployment_exporter.js

```diff
@@ -49,7 +49,7 @@ function checkForErrors(deployment) {
 }
 
 function setUpDeploymentStructure(deployment) {
-  const cleanDeployment = JSON.parse(JSON.stringify(deployment, null, 2));
+  const cleanDeployment = JSON.parse(JSON.stringify(deployment, null, 2).concat('\n'));
```

Comment:
The `cleanDeployment` name wasn't great... let's just remove the variable which is only used once and do l.54:
`deploymentToExport[GENERATOR_NAME] = JSON.parse(JSON.stringify(deployment, null, 2).concat('\n'));`